video/out/opengl/egl_helpers.c

```diff
@@ -332,7 +337,7 @@ EGLDisplay mpegl_get_display(EGLenum platform, const char *platform_ext_name,
     }
 
     // (It should be impossible to be missing, but uh.)
-    if (!gl_check_extension(exts, "EGL_EXT_client_extensions"))
+    if (!gl_check_extension(exts, "EGL_EXT_platform_base"))
```

Comment:
Please drop the comment - it was referring to the old "missing" client extension, which was wrong.

The rest of the patch seems good. Even though it reuses the (somewhat bonkers) `mpegl_get_display` it will do in the short term.